packages/neon_framework/packages/neon_http_client/lib/src/interceptors/authentication_interceptor.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:http/http.dart';
+import 'package:meta/meta.dart';
+import 'package:neon_http_client/src/interceptors/http_interceptor.dart';
+import 'package:nextcloud/nextcloud.dart';
+
+@internal
+final class AuthenticationInterceptor implements HttpInterceptor {
+  var _blocked = false;
+
+  @override
+  bool shouldInterceptRequest(BaseRequest request) {
```

Comment:
I think this interceptor should only intercept requests to the nextcloud server. Please match the baseURL